html-syntax/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>My Favorite Pokemon</title>
+  </head>
+  <body>
+    <img src="https://www.pngmart.com/files/22/Zapdos-Pokemon-PNG-Pic.png">
```

Comment:
Great job, Ben!
One small request: would you please add `alt` attribute to `img` tag?